architecture/azure/resources.adoc

```diff
@@ -0,0 +1,106 @@
+[[azure-resources]]
+= Resources
+//openshift-aro 3.11 specific
+
+This section describes all azure resources used to build a cluster.
+
+== Virtual Machine Scale Sets
+
+=== ss-masters
+Scale set of 3 virtual machines used for core services.
+Etcd daemon and the OpenShift API Server run here among other core services.
+The ARO Sync Pod also runs on one of the Master Virtual Machines.
+
+=== ss-infra-TIMESTAMP
+Scale set of 3 virtual machines used for running infrastructure pods.
+Among other services running here are:
+
+- the docker registry
+- routers for applications running on openshift
+- OpenShift web console
+- template service broker
+- cluster monitoring Prometheus (used by ARO SREs to monitor cluster health)
+
+=== ss-compute-TIMESTAMP
+Scale set of virtual machines used by customers' pods. All application pods
+and build pods are scheduled on nodes within this scale set.
+The number of virtual machines can be adjusted by the customer.
+
+== Disks
+All *kubernetes-dynamic-pvc-&#42;* Disks are attached to their respective
+Virtual Machine for azure-disk class PVs.
+Please note that PVs are wiped during normal cluster operation when a
```

Comment:
Is this accurate?  Why would PVs be wiped during upgrades?  From my understanding is that any `emptrydir` storage option would fit into this but any PV would persist.  The drain/stop of a node would cause the PV to migrate to a new host.

Reply:
Removed the sentence about  PVs getting wiped. Got them confused with emptydir.